src/utils/looper.ts

```diff
@@ -1,22 +1,28 @@
 import { processProbeStatus } from './process-server-status'
 import { Config } from '../interfaces/config'
 import console, { log } from 'console'
-import { probing } from '../utils/probing'
+import { probing } from './probing'
 import { validateResponse, sendAlerts } from './alert'
+import { probeLog } from './logger'
+import { AxiosResponseWithExtraData } from '../interfaces/request'
 
 const MILLISECONDS = 1000
 
-async function doProbes(config: Config) {
+/**
+ * doProbe sends out the http request
+ * @param {object} config contains all the configs
+ */
+async function doProbe(config: Config) {
   // probe each url
   log('\nProbing....')
+  let probRes: AxiosResponseWithExtraData
```

Comment:
Why we're defining the variable in here?
What is the problem with previous `const` variable?

Reply:
ok. thanks!